app/src/components/Tabs/my-files-tab.tsx

```diff
@@ -416,7 +418,7 @@ const MyFilesTab: FC<MyFilesTabProps> = ({
                                         ></IconButton>
                                       </PopoverTrigger>
                                       <PopoverContent
-                                        h="128px"
+                                        h="176px"
```

Comment:
Hardcoded dimensions aren't great when working with different languages in i18n. Layouts break too easily when some text is shorter or longer (see e.g. the cards on the dashboard when switching to Spanish).